package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hero-ui",
-  "version": "4.0.10",
+  "version": "4.0.11",
```

Comment:
Might run in to problems with this, might be worth doing a separate commit to `master` right before release since buildkite will look at `master` and and hook on to a commit message that matches a version number. See the new section in README